eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -37,15 +37,17 @@ jobs:
 
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'linux_musl_x64') }}:
-      - ${{ if or(eq(parameters.jobParameters.isExtraPlatformsBuild, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
-        - (Alpine.edge.Amd64.Open)Ubuntu.2204.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-edge-helix-amd64
+      # Alpine edge builds should only be used in main
+      - ${{ if eq(contains(coalesce(variables['System.PullRequest.TargetBranch'], variables['Build.SourceBranch'], ''), 'main'), 'true') }}:
```

Comment:
I do not see the alpine edge running in outer loop, so this condition is probably not right.

@jkoritzinsky Do you happen to know how to check for main vs. servicing here?

Reply:
Good call on checking. I did not do that.

Reply:
I was thinking about doing the same thing for Fedora, just testing it in `main`. It would be nice to just have one fast-moving distro in servicing to maintain. We get significant RHEL-family coverage from CentOS Stream and Alma Linux. I think that + Fedora in `main` is a good model.

Reply:
I just realized. It's because the condition is extra platforms not outerloop.